docs/getting-started.md

```diff
@@ -149,7 +149,7 @@ And open http://localhost:8080 in a browser. You should see a blue “Hello Worl
 Now that you have webpack configured to compile Sass into CSS, let's include the Sass files for the Material Design button. First install the Node dependency:
 
 ```
-npm install --save-dev @material/button
```

Comment:
Don't remove `--save-dev`. I'm pretty sure I stopped someone from doing this before (or had them revert it).

Reply:
Why are we telling people to add their non-build step deps to devDeps?  `dependencies` are modules which are also required at runtime.